modules/protect/shared-functions.php

```diff
@@ -211,8 +211,8 @@ function jetpack_clean_ip( $ip ) {
 		$ip = $matches[1];
 	}
 
-	if ( function_exists( 'parse_url' ) ) {
-		$parsed_url = parse_url( $ip );
+	if ( function_exists( 'wp_parse_url' ) ) {
```

Comment:
Do we need this `function_exists`? Seems like we support only newer versions of WP that already should have it defined.

Reply:
That's a valid question. `parse_url` should have also always been available, I'd imagine.